spec/apis/register-application.spec.js

```diff
@@ -19,7 +19,7 @@ describe("registerApplication", function() {
   describe(`Application name errors`, () => {
     it(`should throw an error if the name isn't a non empty string`, () => {
       expect(() => {
-        singleSpa.registerApplication(app);
+        singleSpa.registerApplication(null);
```

Comment:
Changed usage of `app` as a wrong argument, because object arguments triggers object configuration validations